src/dropdown/dropdown.component.ts

```diff
@@ -345,7 +357,7 @@ export class Dropdown implements OnInit, AfterContentInit, OnDestroy {
 		let selected = this.view.getSelected();
 		if (selected && !this.displayValue) {
 			if (this.type === "multi") {
-				return of(`${selected.length} ${this.selectedLabel}`);
+				return of(`${this.selectedLabel}`);
```

Comment:
this whole method (`getDisplayValue`) should be something like:

```ts
    /**
     * Returns the display value if there is a selection and displayValue is set,
     * if there is just a selection the ListItem content property will be returned,
     * otherwise the placeholder will be returned.
     */
    getDisplayValue(): Observable<string> {
        if (!this.view) {
            return;
        }
        let selected = this.view.getSelected();
        if (selected && !this.displayValue) {
            if (this.type === "multi") {
                return of(this.placeholder);
            } else {
                return of(selected[0].content);
            }
        } else if (selected) {
            return of(this.displayValue);
        }
        return of(this.placeholder);
    }
```

This always returns the placeholder value when we're multi-selecting (unless we have a displayValue, which always overrides)

Reply:
to be clear, the big change here is updating `return of(`${this.selectedLabel}`);` to be `return of(this.placeholder);` and updating the comment to be more descriptive :+1: